### PR TITLE
Feature cut for F411 and F722

### PR DESCRIPTION
**Pull request by DzikuVx**
* no CLI help -> 3kB
* SPEKTRUM and SRLX telemetry -> 2kB
* SmartPort Master -> 1.5kB

### CODE REVIEW

File: src/main/target/common.h [hunk 1]
```diff
@@ -202,4 +202,11 @@
 #elif !defined(STM32F7)
 #define MAX_MIXER_PROFILE_COUNT 1
 #endif
+
+#if (MCU_FLASH_SIZE <= 512)
+    #define SKIP_CLI_COMMAND_HELP
+    #undef USE_SERIALRX_SPEKTRUM
```
Comment (mmosca) [inline src/main/target/common.h:208]: Probably cleaner to move the defines to inside the MCU_FLASH_SIZE > 512 above, instead of using #undef

**Comment by mmosca** [comment]
If you look at common_post.h there are some lines commented out that could remove some of the less common magnetometers as well,  unless flash is > 512

File: src/main/target/common.h (hunk 1 above)
Comment (DzikuVx) [reply src/main/target/common.h:208]: i seen that but elif makes it very hard to read. Probably will clean the whole section

Comment (mmosca) [reply src/main/target/common.h:208]: Maybe move the elif check to common_post and check for f411, instead of not f7

**Pull request merged**